src/runtime_src/core/edge/user/device_linux.cpp

```diff
@@ -159,6 +173,424 @@ struct kds_cu_info
   }
 };
 
+struct aie_reg_read
+{
+  using result_type = query::aie_reg_read::result_type;
+
+  static result_type
+  get(const xrt_core::device* device, key_type key, const boost::any& r, const boost::any& c, const boost::any& reg)
+  {
+    auto dev = get_edgedev(device);
+    uint32_t val;
+    auto row = boost::any_cast<int>(r);
+    auto col = boost::any_cast<int>(c);
+    auto v = boost::any_cast<std::string>(reg);
+    
+#ifdef XRT_ENABLE_AIE
+#ifndef __AIESIM__
+    std::map<std::string , uint32_t> regmap = {
```

Comment:
no space between string and ','

Reply:
done

Reply:
`const std::map<std::string , uint32_t> regmap = {`

Reply:
already fixed